docs/core/entity/number.md

```diff
@@ -19,7 +19,7 @@ Properties should always only return information from memory and not do I/O (lik
 | native_min_value | float | 0 | The minimum accepted value in the number's `native_unit_of_measurement` (inclusive)
 | native_step | float | **See below** | Defines the resolution of the values, i.e. the smallest increment or decrement in the number's | native_unit_of_measurement | string | `None` | The unit of measurement that the number's value is expressed in. If the `native_unit_of_measurement` is °C or °F, and its `device_class` is temperature, the number's `unit_of_measurement` will be the preferred temperature unit configured by the user and the number's `state` will be the `native_value` after an optional unit conversion.
 | native_value | float | **Required** | The value of the number in the number's `native_unit_of_measurement`.
-`native_unit_of_measurement`
+| native_unit_of_measurement | string | None | The unit of measurement that the sensor's value is expressed in. If the `native_unit_of_measurement` is °C or °F, and its `device_class` is temperature, the sensor's `unit_of_measurement` will be the preferred temperature unit configured by the user and the sensor's `state` will be the `native_value` after an optional unit conversion. If not `None`, the sensor is assumed to be numerical and will be displayed as a line-chart in the frontend instead of as discrete values.
```

Comment:
> If not `None`, the sensor is assumed to be numerical and will be displayed as a line-chart in the frontend instead of as discrete values.

That part not longer true and should be removed. Even without a valid unit of measurement, charts can be displayed (for example, when having a valid state class).

Reply:
I'll pull that out when I get to a development machine later today. Should the `sensor` wording be updated as well since this verbiage was copied from there? Potentially in another PR? 

https://github.com/home-assistant/developers.home-assistant/blob/c5eae6f4e5f044c0b0ef859782a7811ad30b9624/docs/core/entity/sensor.md?plain=1#L19

Reply:
Yeah `sensor` is also incorrect in this case (probably history).

Reply:
@frenck I removed the sentence surrounding the current behavior per the feedback. Also, I included the `sensor` update in this PR and updated the title since it's a relatively small change about the same topic.